lib/driver.js

```diff
@@ -323,6 +323,30 @@ class AndroidDriver extends BaseDriver {
     this.sessionChromedrivers[CHROMIUM_WIN] = this.chromedriver;
     this.proxyReqRes = this.chromedriver.proxyReq.bind(this.chromedriver);
     this.jwpProxyActive = true;
+
+    // Dismiss Chrome welcome dialog
+    if (opts.chromeOptions.args.indexOf('--no-first-run') !== -1) {
+      await this.dismissChromeWelcome();
+    }
+  }
+
+  async dismissChromeWelcome () {
+    log.info("Trying to dismiss Chrome welcome");
+    let activity = await this.getCurrentActivity();
+    if (activity === "org.chromium.chrome.browser.firstrun.FirstRunActivity") {
+      let el = await this.findElOrEls('id', 'com.android.chrome:id/terms_accept', false);
+      await this.click(el.ELEMENT);
+      el = await this.findElOrEls('id', 'com.android.chrome:id/negative_button', false).catch((e) => {
```

Comment:
rather than mixing promises in, let's use try/catch:

```js
try {
  el = await this.findElOrEls(...);
} catch (e) {
  log.warn(`...`);
  return;
}
await this.click(el.ELEMENT);
```